frontend_app_admin_workspace_user/src/index.js

```diff
@@ -13,6 +13,7 @@ const appInterface = {
   renderAppFullscreen: data => {
     document.getElementById(data.config.domContainer).classList.add('fullWidthFullHeight')
 
+    // INFO - CH - 2020-01-11 - User Router instead of BrowserRouter because BrowserRouter doesn't have a history props
```

Comment:
At `/index.dev.js` you used `const history = require('history').createBrowserHistory()` can't you do the same here?

```suggestion
    // INFO - CH - 2020-01-11 - Use Router instead of BrowserRouter because BrowserRouter doesn't have a history props
```

Reply:
no, I use createHistory in index.dev.js because I don't have the history that comes from the frontend.
Which index.js has

Reply:
will fix the typo though

Reply:
typo fixed